### DISCUSSION TITLE
Nozzle is to high?

### DISCUSSION THREAD

**Discussion by olskar** [Jyers/Marlin#1016] [Q&A] [answered]
Hello!
Not sure how I understand how this works, but after making a mesh and setting the z offset, the nozzle is to high when printing?

I notice during printing the first layer Z is at 0.20, isnt this supposed to be at 0? Can that  be the problem?

![IMG_20210616_125229](https://user-images.githubusercontent.com/18314560/122207080-05aa4200-cea2-11eb-8d18-199f072a4320.jpg)

**Comment by msdossajo** [comment]
0.2 is the first layer, if printing @0.2mm layer, that's the gap that will be filled with material. 
The Z offset will change according with the Bltouch (or similar) position, in my case mine after re-installing the Bltouch is set to -2.18, but before was around -1.2.

**Reply by gothcha** [reply]
The Z offset is currently showing -1 this is very low, with the standard mount mine is -3.13. Use the Z offset tool and remember to store the settings after finding the correct value.

**Reply by pderbyshire** [reply]
Yeah -1.00 z offset probably isn't enough, my mounts puts my BLTouch at -2.62 to be level, you can even microstep it down during a print via the Tune menu but you should try and get it closer to the bed before doing so first.

**Reply by olskar** [reply]
Weird, this is my nozzle at z offset -1 :/

![IMG_20210616_155420](https://user-images.githubusercontent.com/18314560/122234906-89bdf300-cebd-11eb-9a56-42fe513f978d.jpg)

**Reply by pderbyshire** [reply]
Looks like your BLTouch sits much lower than mine does, everyone's setups a little different.

**Comment by Jyers** [comment, marked as answer]
The best approach for getting the right z offset is to use the step up/down buttons to tune it while printing, that way you get the offset that results in the best quality first layer.
